pkg/report/impact_score.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2025 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package report
+
+import (
+	"github.com/google/syzkaller/pkg/report/crash"
+)
+
+// impactOrder represent an ordering of bug impact severity. The earlier
+// entries are considered more severe.
+var impactOrder = []crash.Type{
+	crash.KASANWrite,
+	crash.KASANUseAfterFreeWrite,
+	crash.KASANInvalidFree,
+	crash.KASANUseAfterFreeRead,
+	crash.KASANRead,
+	crash.KASANOther,
```

Comment:
We don't actually need category Other here. What we need is "unknown" that will not be scored here and reported as -1 to highlight the problem.
#6154 to fix it.